compiler/test/dotty/tools/utils.scala

```diff
@@ -47,7 +47,11 @@ def assertThrows[T <: Throwable: ClassTag](p: T => Boolean)(body: => Any): Unit
 end assertThrows
 
 def toolArgsFor(files: List[JPath], charset: Charset = UTF_8): List[String] =
-  files.flatMap(path => toolArgsParse(Files.lines(path, charset).limit(10).toScala(List)))
+  files.flatMap(path => toolArgsParse {
+    val stream = Files.lines(path, charset)
+    try stream.limit(10).toScala(List)
+    finally stream.close()
+  })
```

Comment:
We normally use scala.util.Using for this:
```suggestion
  files.flatMap(path => toolArgsParse(Using(Files.lines(path, charset))(_.limit(10).toScala(List))))
```